--- a/pkg/cav-linux-dkms/opt/COMODO/load_cmgdaemon.sh	2013-02-25 09:50:48.000000000 +0100
+++ b/pkg/cav-linux-dkms/opt/COMODO/load_cmgdaemon.sh	2021-12-11 12:20:09.819474376 +0100
@@ -36,7 +36,7 @@
 fi
 
 FOUND=0
-for TOS in Red Fedora SUSE Debian Ubuntu SliTaz Mint
+for TOS in Red Fedora SUSE Debian Ubuntu SliTaz Mint Manjaro Arch
 do
  if [ $FOUND -eq 0 ]
  then
@@ -58,10 +58,7 @@
 Debian|Ubuntu|Mint)
   . /lib/lsb/init-functions
   ;;
-SliTaz)
-  ;;
 *)
-  . /etc/init.d/functions
   ;;
 esac
 
@@ -99,7 +96,7 @@
         ;;
     *)
         cd $CHGDIR
-        daemon $PROG_NAME
+        $PROG_NAME
         ;;
     esac 
        
@@ -130,7 +127,7 @@
         killproc $PROG_NAME
         rc_status -v
         ;;
-    Debian|Ubuntu|Mint)
+    Debian|Ubuntu|Mint|Manjaro|Arch)
         pkill -KILL cmgdaemon >/dev/null 2>&1
         ;;
     SliTaz)
